AnnoDesigner/App.xaml.cs

```diff
@@ -124,7 +127,7 @@ private async void Application_Startup(object sender, StartupEventArgs e)
 
                         if (!createdNewMutex)
                         {
-                            MessageBox.Show("Another instance of the app is already running.");
+                            _messageBoxService.ShowMessage("Another instance of the app is already running.", string.Empty);
```

Comment:
Feels a bit odd to not title the `MessageBox` - could we maybe give `title` a default (make it optional), or provide a title here?
eg
* For `ShowError`, the default could be "Error"
* `ShowQuestion`, "Question"
... etc

Reply:
I will add the following default titles to the implementation/extensions.

Method | Default Title/Header
------------ | -------------
`ShowMessage` | Information
`ShowError` | Error
`ShowQuestion` | Question
`ShowWarning` | Warning